src/findings/finding_container.py

```diff
@@ -49,6 +49,27 @@ def getAllFindings(cls):
     def getActionableFindings(cls):
         return [finding for finding in cls._finding_results if finding.actionable]
 
+    @classmethod
+    def toHumanReadableMessage(cls):
+        output_message = ""
+        file_to_findings = {}
+        for finding in cls.getActionableFindings():
+            # Create a map to summarize the findings based on proto file name.
+            if finding.location.proto_file_name not in file_to_findings:
+                file_to_findings[finding.location.proto_file_name] = []
+            file_to_findings[finding.location.proto_file_name].append(finding)
+        # Add each finding to the output message.
+        for file_name in file_to_findings:
+            # Customize sort key function to output the findings in the same
+            # file based on the source code line number.
+            def _get_line_number(finding):
+                return finding.location.source_code_line
+
+            file_to_findings[file_name].sort(key=_get_line_number)
```

Comment:
For a single line key function, it's more compact to just use a lambda.
```python
file_to_findings[file_name].sort(key=lambda f: f.location.source_code_line)
```

Reply:
lamda function in python is new to me, thanks!